package.json

```diff
@@ -2,34 +2,27 @@
   "name": "empty-dir",
   "description": "Check if a directory is empty.",
   "version": "0.1.0",
-  "homepage": "https://github.com/tkellen/node-empty-dir",
-  "author": {
-    "name": "Tyler Kellen",
-    "url": "http://goingslowly.com/"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/tkellen/node-empty-dir.git"
-  },
+  "homepage": "https://github.com/tkellen/js-empty-dir",
+  "author": "Tyler Kellen <http://goingslowly.com/>",
+  "repository": "tkellen/js-empty-dir",
   "bugs": {
-    "url": "https://github.com/tkellen/node-empty-dir/issues"
+    "url": "https://github.com/tkellen/js-empty-dir/issues"
   },
-  "licenses": [
-    {
-      "type": "MIT",
-      "url": "https://github.com/tkellen/node-empty-dir/blob/master/LICENSE"
-    }
-  ],
+  "license": "MIT",
   "main": "index.js",
+  "files": ["index.js"],
   "engines": {
     "node": ">= 0.8.0"
   },
   "scripts": {
-    "test": "mocha -R spec test"
+    "test": "mocha"
+  },
+  "dependencies": {
+    "try-open": "^0.1.0"
   },
   "devDependencies": {
-    "chai": "~1.9.1",
-    "mocha": "~1.18.2"
+    "chai": "*",
```

Comment:
bah, sorry I ran an "updater" by accident on package.json, I meant to revert that (and thought I did, so I must have run it twice! fwiw, the reason the `updater` removed `mocha` is b/c it's not `require`d anywhere in the project. The `chai` wildcard is a hard-coded default I added a long time ago for a reason I can no longer remember.

anyway, I have two strong habits that are hard _not to do_ before I push up: run `update` and run `deps` (for lint-deps). but I try to only do this on my own projects though. so again, apologies.